packages/pigment-react/tests/README.md

```diff
@@ -1,3 +1,33 @@
-# Adding new fixtures
+## Folder structure
 
-Create a new file name with `[name].input.js` and add `styled`, `css` or other zero-runtime calls into the file. Also add equivalent `[name].output.js` and `[name].output.css` and run the test. After the new test fails, get the results from the received output and add it to the equivalent js and css files. This is equivalent to snapshot testing and will make sure any change in internal css generation logic does not fail any other existing tests.
+- `tests` is the root folder for all tests
+  - `fixtures` contains all the fixtures for the tests
+    - `*.input.js` are the input files created manually
+    - `*.output.*` are the expected output files created by running the tests
+  - `*.test.js` are the test files that run the fixtures
+
+## Running tests
+
+At the root project terminal:
+
+```bash
+pnpm --filter @pigment-css/react run test
```

Comment:
The correct command would be `pnpm nx run @pigment-css/react:test`. This'll make sure to also build the code if it isn't built already. Same for `test:update`

`pnpm nx run @pigment-css/react:test:update`